### PR TITLE
Update plex-preferences.sh

### PR DESCRIPTION
**Pull request by someone-stole-my-name**
**Description of the change**

Create `Preferences.xml` on startup if it doesn't exist & remove directory creation from entrypoint.

**Benefits**

Fixes `PLEX_PREFERENCE_X` not being applied on startup.

**Possible drawbacks**

**Applicable issues**

**Additional information**

### CODE REVIEW

**Comment by onedr0p** [comment]
Hi @someone-stole-my-name 👋🏼  I think you change makes this line of code obsolete.

https://github.com/k8s-at-home/container-images/blob/main/apps/plex/entrypoint.sh#L13-L14

But however I think `mkdir -p "$(dirname "${prefFile}")"` should exists right below the `prefFile` definition in `plex-preferences.sh`

**Comment by someone-stole-my-name** [comment]
Hey @onedr0p, thanks for reviewing the PR. Fixed and squashed.

File: apps/plex/shim/plex-preferences.sh [hunk 1]
```diff
@@ -28,6 +28,16 @@ function setPref {
 
 prefFile="${PLEX_MEDIA_SERVER_APPLICATION_SUPPORT_DIR}/Plex Media Server/Preferences.xml"
 
+# Create empty shell pref file if it doesn't exist already
+if [ ! -e "${prefFile}" ]; then
```
Comment (onedr0p) [inline apps/plex/shim/plex-preferences.sh:32]: Wouldn't `! -f` be more accurate?

Comment (someone-stole-my-name) [reply apps/plex/shim/plex-preferences.sh:32]: Shamelessly stolen from upstream file https://github.com/plexinc/pms-docker/blob/master/root/etc/cont-init.d/40-plex-first-run#L79

Comment (onedr0p) [reply apps/plex/shim/plex-preferences.sh:32]: oh interesting! If it works for them it works for us lol

**Review by onedr0p — APPROVED** [review]

**Pull request merged**